doc/parameter.rst

```diff
@@ -16,6 +16,13 @@ Before running XGBoost, we must set three types of parameters: general parameter
   :backlinks: none
   :local:
 
+********************
+Global Configuration
+********************
+The following parameters can be set in the global scope, using ``xgb.config_context()`` (Python) or ``xgb.set.config()`` (R).
+
+* ``verbosity``: Verbosity of printing messages. Valid values of 0 (silent), 1 (warning), 2 (info), and 3 (debug).
```

Comment:
Should we deprecate the one used in training?

Reply:
Yes. How soon should we do so?

Reply:
Not sure.  We can discuss it for the next release when stuffs like rmm and gpu_id are ported here.